ansible/roles/jenkins-worker/tasks/main.yml

```diff
@@ -19,6 +19,16 @@
     state: present
     line: ::1 localhost.localdomain localhost
 
+- name: create NODE_TEST_DIR directory
+  file: path=/home/{{ server_user }}/tmp state=directory
+
+- name: set NODE_TEST_DIR permission and owner
+  file:
+    path: /home/{{ server_user }}/tmp
+    owner: "{{ server_user }}"
+    group: "{{ server_user }}"
+    mode: 0655
```

Comment:
I think it should be 0755 so that iojs can cd into it (see [StackOverflow](https://unix.stackexchange.com/questions/150449/what-does-execute-permission-on-a-folder-means#)). Not entirely sure though.